app/views/users/show.html.erb

```diff
@@ -1,24 +1,136 @@
-<p class="fs-5 mt-4 mb-2">
-  <strong>Név:</strong>
-  <%= @user.name %>
-</p>
+<div class="row">
+  <div class="col-12">
+    <div class="card">
+      <div class="card-header text-center">
+        <span class="text-muted">
+          <svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" fill="currentColor" class="bi bi-person-circle" viewBox="0 0 16 16">
+          <path d="M11 6a3 3 0 1 1-6 0 3 3 0 0 1 6 0z"/>
+          <path fill-rule="evenodd" d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8zm8-7a7 7 0 0 0-5.468 11.37C3.242 11.226 4.805 10 8 10s4.757 1.225 5.468 2.37A7 7 0 0 0 8 1z"/>
+        </svg>
+        </span>
+      </div>
+      <div class="card-body text-center row justify-content-lg-center justify-content-start">
+        <div class="col-auto text-start">
+          <p>
+            <strong>Név: </strong><%= @user.name %>
+          </p>
+          <p>
+            <strong>Email: </strong><%= @user.email %>
+          </p>
+          <% if admin_signed_in? %>
+            <p>
+              <strong>Provider: </strong><%= @user.provider %>
+            </p>
+            <p>
+              <strong>Admin: </strong><%= @user.admin %>
```

Comment:
Please use non-breakable space (`&nbsp;`) AFTER all the `</strong>` tags, and remove the simple spaces before them.